wp/plugin/Makefile

```diff
@@ -97,15 +65,8 @@ test: test.unit
 
 .PHONY: test.unit
 test.unit: install
-	ocamlbuild -r -use-ocamlfind \
-		-pkgs $(TEST_PKGS) \
-		-tag $(TEST_TAG) -Is $(TEST_Is_UNIT) test.native
-	./test.native
+	dune runtest tests/unit
```

Comment:
I suggest adding `--force` here, so that if bap is changed the tests are still be running (the default behavior of dune is to skip test if their dependencies wasn't changed)